packages/styletron-core/src/index.js

```diff
@@ -6,15 +6,17 @@ class StyletronCore {
 
   /**
    * Create a new StyletronCore instance
-   * @param {object} [opts]           An object containing options
-   * @param {string} [opts.prefix=''] A prefix for generated CSS class names
+   * @param {object} [opts]                           An object containing options
+   * @param {string} [opts.prefix='']                 A prefix for generated CSS class names
+   * @param {boolean} [opts.readableClassNames=false] Use readable class names
    */
-  constructor({prefix = ''} = {}) {
+  constructor({prefix = '', readableClassNames = false} = {}) {
```

Comment:
What about something like `compressedClassNames = true`. Might be more clear than trying to describe the "readable" class name form.